dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -107,8 +107,8 @@ export default defineComponent({
         | { [key in DataTypeEnum]: AggregatedFrameworkDataSummary }
         | undefined,
       FRAMEWORKS_WITH_VIEW_PAGE,
-      isUserCompanyOwner: false,
-      isUserUploader: false,
+      isUserCompanyOwnerOrUploader: false,
+      isUserDatalandUploader: false,
```

Comment:
maybe isUserKeycloakUploader to make the keycloak role more explicit?

Reply:
Ja stimmt ist besser. Habe es angepasst.